pyproject.toml

```diff
@@ -0,0 +1,72 @@
+[tool.poetry]
+name = "my-package"
+version = "0.1.0"
+description = "A short description of the package."
+license = "MIT"
+authors = [
+    "PDA-FGA <rocha.carla@gmail.com>"
+]
+maintainers = [
+    "Bruno-Felix <auxbfelix@gmail.com>"
```

Comment:
We need to add all the maintainer here, we have a list with user and email of them, what do you think to add here?